Topic-2-CSS/Task 1/Css/styles.css

```diff
@@ -0,0 +1,13 @@
+header, footer, aside, nav{
+  background-color: red;
+}
+
+html *
```

Comment:
The * selector isn't needed because fonts properties are always inherited by children unless you define another style that changes it.